tests/apps_test.go

```diff
@@ -51,7 +51,7 @@ var _ = Describe("deis apps", func() {
 
 			Specify("that user cannot get information about that app", func() {
 				sess, err := cmd.Start("deis info -a %s", &user, bogusAppName)
-				Eventually(sess.Err).Should(Say("Not found."))
+				Eventually(sess.Err).Should(Say("Error: Not Found"))
```

Comment:
Understanding this PR just followed the precedent before of repeated/duplicated expected cli errors, wondering what your thoughts are on creating re-usable vars (perhaps in https://github.com/deis/workflow-e2e/blob/master/tests/settings/settings.go or maybe elsewhere) for the most commonly expected error outputs.  

For instance, `"Error: Not Found"` could be the value for `var ErrNotFound` and then it'd just need to be changed in one spot when/if the cli error changes.  Just a thought and def not blocking for this PR...